File: python/d069.py
# -*- coding: utf-8 -*-
"""
Created on Mon Jan  6 12:35:18 2020

@author: Username
"""

import sys

for n in [input()]:
    n=n.replace('\n','')
    for i in range(int(n)):
        num=int(input())
        if num%400 == 0 or num%4 == 0 and num% 100 !=0:
            print('a leap year')
        else: print('a normal year')

File: c++/d069.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0); cout.tie(0);
    int t; cin >> t;
    while (t--)
    {
        int num; cin >> num;
        if (!(num%400) || !(num%4) && num%100) cout << "a leap year\n";
        else cout << "a normal year\n";
    }

}
